packages/react-reconciler/inline.art.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// This file intentionally does *not* have the Flow annotation.
+// Don't add it. See `./inline-typed.js` for an explanation.
```

Comment:
Worth adding some automated check for? Or do you think the inline comment will be sufficient (when we create new renderers)?

Reply:
If you add it Flow will fail so we won’t do it accidentally. I just left a comment so somebody doesn’t think we *should* add it as part of normal flowification process and then get confused.